src/frontend/src/icp/components/tokens/IcAddTokenForm.svelte

```diff
@@ -8,33 +7,27 @@
 	export let indexCanisterId = '';
 </script>
 
-<p class="mb-2 mt-1">{replaceOisyPlaceholders($i18n.tokens.import.text.info)}</p>
-
-<p class="mb-4 font-bold text-brand-primary">
-	<ExternalLink
-		href="https://github.com/dfinity/oisy-wallet/blob/main/HOW-TO.md#custom-icrc-token-integration"
-		ariaLabel={$i18n.tokens.import.text.open_github_howto}
-	>
-		{$i18n.tokens.import.text.github_howto}
-	</ExternalLink>
-</p>
-
 <div class="stretch">
+	<p class="mb-6">{$i18n.tokens.import.text.info}</p>
+
 	<label for="ledgerCanisterId" class="px-4.5 font-bold"
-		>{$i18n.tokens.import.text.ledger_canister_id}:</label
+		>{$i18n.tokens.import.text.ledger_canister_id}: <span class="text-blue-ribbon">*</span></label
```

Comment:
shouldn't the asterisk go before the dots?

`Ledger Canister*:`

Reply:
Not according Figma.
![Uploading Capture d’écran 2024-11-13 à 19.38.25.png…]()

Reply:
cool! tks for checking!